OpenRA.Mods.Common/Traits/ThrowsShrapnel.cs

```diff
@@ -19,8 +19,14 @@ public class ThrowsShrapnelInfo : ITraitInfo
 	{
 		[WeaponReference, FieldLoader.Require]
 		public string[] Weapons = { };
+
+		[Desc("Don't fire all weapons at once, but choose one by chance.")]
+		public bool RandomWeapon = false;
```

Comment:
The weapons should have been random originally. It makes no sense to fire more than one weapon as shrapnel at a time, IMO.